README.md

```diff
@@ -152,14 +152,16 @@ These values can be found in the [PAYONE Merchant Interface](https://pmi.pay1.de
+`TENANT1_SECURE_KEY`                  | Yes                | if provided and not empty, the value is used as the key for decrypting data from fields "IBAN" and "BIC" for payments with CustomType "PAYMENT_BANK_TRANSFER". The data must be the result of a Blowfish ECB encryption with said key and encoded in HEX.                                                                                                                                                                                                                  | "" (empty String)
+`TENANT1_UPDATE_ORDER_PAYMENT_STATE`  | Yes                | if _true_ - `Order#paymentState` will be updated when payment status notification is received from Payone. By default the order's state remains unchanged. See [Order Payment Status Mapping](/docs/Order-Payment-Status-Mapping.md) for more details.                                                                                                                                                                                                                     | "false"
+`HIDE_CUSTOMER_PERSONAL_DATA`         | No                 | if _true_ - all customer related personal data will be replaced by `<HIDDEN>` placeholder in all logs. If _false_ - all customer related personal data will be visible in logs.                                                                                                                                                                                                                                                                                            | "true"
+`LOG_LEVEL`                           | No                 | log-level for [service logging](http://logback.qos.ch/manual/architecture.html#effectiveLevel)                                                                                                                                                                                                                                                                                                                                                                             | "INFO"
+`CT_AUTH_URL`                         | Yes                | if set, this value will be used for authentication by the sphere client.                                                                                                                                                                                                                                                                                                                                                                                                   | "https://auth.europe-west1.gcp.commercetools.com"
```

Comment:
Thanks for your PR. I guess you missed a prefix here since this service is multi-tenant which means that it should be possible to configure AUTH and API URLs in a tenant-specific manner?

Reply:
yes, i have corrected the naming accordingly and placed them after the other tenant attributes now.